starter-code/pizza.js

```diff
@@ -65,21 +92,58 @@ function renderButtons() {
 
 function renderPrice() {
   // Iteration 4: change the HTML of `<aside class="panel price">`
+  /* state.forEach((elem, i) => {
+      if (elem) {
+        document.querySelectorAll(".panel price > ul")[i].style.visibility =
+          "visible";
+      } else {
+        document.querySelectorAll(".panel price > ul")[i].style.visibility =
+          "hidden";
+      }
+    }); */
+  let newSum = "";
+  let sum = basePrice;
+  for (const element in state) {
+    if (state[element]) {
+      sum += ingredients[element].price;
+      newSum += `<li>${ingredients[element].price} ${ingredients[element].name}</li>`;
+    }
+  }
+  document.querySelector("strong").innerText = `$${sum}`;
+  document.querySelector(".price ul").innerHTML = newSum;
 }
 
-
-renderEverything()
+renderEverything();
 
 // Iteration 1: Example of a click event listener on `<button class="btn btn-pepperonni">`
-document.querySelector('.btn.btn-pepperonni').onclick = function() {
-  state.pepperonni = !state.pepperonni
-  renderEverything()
-}
+document.querySelector(".btn.btn-pepperonni").onclick = function(event) {
+  event.target.classList.toggle("active");
```

Comment:
cool that you did it here, but you can also do it one for everything in the function renderButton() function above :) Will save you a lot of lines and makes the code cleaner

Reply:
Other than that 👍 👍 !!